pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+
+
```

Comment:
[nit]: we don't need these two lines here. We can just have the pre-submission checklist header be the first line here:

```suggestion
```